### PR TITLE
[NCL-6018] Split log lines across multiple items

### PR DESCRIPTION
**Pull request by brusdev**
Split log lines across multiple items to fix NotYetAlignedFromDependencyTree.

### CODE REVIEW

**Comment by thescouser89** [comment]
LGTM!

File: pig/src/main/java/org/jboss/pnc/bacon/pig/impl/pnc/PncBuild.java [hunk 1]
```diff
@@ -88,6 +88,10 @@ public void addBuildLog(String log) {
         buildLog = log == null ? Collections.emptyList() : asList(log.split("\\r?\\n"));
```
Comment (michalszynkiewicz) [inline pig/src/main/java/org/jboss/pnc/bacon/pig/impl/pnc/PncBuild.java:88]: is this method still used?

Comment (brusdev) [reply pig/src/main/java/org/jboss/pnc/bacon/pig/impl/pnc/PncBuild.java:88]: No, ATM it isn't used. Can I remove it or you or do you prefer to keep it for future use?

**Comment by rnc** [comment]
Should this PR not have a test associated with it?

**Pull request merged**